test/pkg/gitea/test.go

```diff
@@ -93,6 +93,7 @@ func TestPR(t *testing.T, topts *TestOpts) (context.Context, func()) {
 	ctx := context.Background()
 	if topts.ParamsRun == nil {
 		runcnx, opts, giteacnx, err := Setup(ctx)
+		runcnx.Clients.Log = runcnx.Clients.Log.With("test", t.Name())
```

Comment:
Since the tests may now be run in parallel, adding the test name is probably necessary to read the logs